data/virt_autotest/host_unattended_installation_files/agama/sle16.jsonnet

```diff
@@ -91,7 +92,17 @@
           echo '{{_SECRET_RSA_PUB_KEY}}' > /root/.ssh/id_rsa.pub
           echo '{{_SECRET_RSA_PUB_KEY}}' >> /root/.ssh/authorized_keys
         |||
-     }
+      }
+    ],
+    init: [
+      {
+        name: "Setup_br0",
+        body: |||
+          zypper --gpg-auto-import-keys ar https://download.opensuse.org/repositories/openSUSE:Factory/standard/ openSUSE:Factory.repo
```

Comment:
Hi @Julie-CAO , 
Is it correct that you just add opensuse repo here as temporary solution, and will switch to use sle16 pkg virt-bridge-setup later(supposed to be available next milestone)?

Reply:
Will the package be included in the default repo? I thought the usage(on https://software.opensuse.org//download.html?project=openSUSE%3AFactory&package=virt-bridge-setup) is the final solution. :open_mouth:

Reply:
Since the package will be added to the default repo around next milestone, I'll not add the repo at here.